### DISCUSSION TITLE
Reference concentration

### DISCUSSION THREAD

**Discussion by shapoo1** [Open-Systems-Pharmacology/Forum#853] [Questions & Problems]
Hi Everyone,
Is it okay to use only Km and Vmax values of different isozymes of an enzyme by keeping the reference concentration as default value of 1umol/l in PK-Sim?

**Comment by StephanSchaller** [comment]
I am not sure I understand 100% what you are trying to do, but if you mean you can e.g. implement a number of isoenzymes with  reference concentration as default value of 1umol/l and use individually characterized Km and Vmax values, sure, you can do this, but you have to be aware, that the effective Vmax is a product of kcat and reference concentration...

**Comment by shapoo1** [comment]
Thank you Stephan,
I am working on an enzyme which has several classes that contribute towards the particular substrate metabolism. I got the required protein content of each class of the enzyme from the litrature. When I tried to put these values in the model, the simulation changes drastically and it doesnt look good. But when I keep the reference concentration of each class as a default value of 1umol/l  the Vmax (kcat) value fit good to the observed data. Also I have selected in -vitro metabolic rate in the presence of recombinant CYPs/ Michaelis-Menten enzyme process so I am using the Km and Kcat value in simulation.

Thanks

**Comment by StephanSchaller** [comment]
If you have selected *in -vitro metabolic rate in the presence of recombinant CYPs/ Michaelis-Menten* you will get specific activity and this needs then to be applied to the actual concentrations of the enzymes.

> required protein content of each class of the enzyme

What type of Data is this. Do you have actual measured PROTEIN levels (for each tissue) or relative tissue mRNA expression with a reference measurement?

**Comment by shapoo1** [comment]
I have the data for the relative protein content of the enzyme in the liver which I got from the literature. The Vmax for the different isozymes are mentioned as per min value in the literature which goes quite well in the simulation. But I am not sure about the reference concentration because it is too high for the simulation

**Comment by StephanSchaller** [comment]
> I have the data for the relative protein content 

So this is not available as an absolute measure such as mol/l?

> The Vmax for the different isozymes are mentioned as per min value in the literature

The *in -vitro metabolic rate in the presence of recombinant CYPs/ Michaelis-Menten* requires Vmax as Amount/time per Amount recombinant Enzyme... how else did you derive kcat?

If you do not have any absolute measures of enzyme amounts a feasible approach is to scale the reference concentration such that it describes your data well. If the default is a good fit, then you are all set.

But this makes it technically a FITTED model and not a bottom-up IVIVE-based prediction.
